ci/build-legacy.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
this is identical to the old `ci/build.sh`, and TC will need to be updated to pick up this file instead

Reply:
yes you are right - I renamed this file back before moving the ci/build.sh to shared-ci